config/initializers/devise.rb

```diff
@@ -131,15 +131,15 @@
   # without confirming their account.
   # Default is 0.days, meaning the user cannot access the website without
   # confirming their account.
-  # config.allow_unconfirmed_access_for = 2.days
+   config.allow_unconfirmed_access_for = 2.days
```

Comment:
Is this overridden by `User#confirmation_required?`?

Reply:
Yes